test/t8_cmesh/t8_gtest_cmesh_face_is_boundary.cxx

```diff
@@ -0,0 +1,188 @@
+/*
+  This file is part of t8code.
+  t8code is a C library to manage a collection (a forest) of multiple
+  connected adaptive space-trees of general element classes in parallel.
+
+  Copyright (C) 2015 the developers
+
+  t8code is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  t8code is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with t8code; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+
+#include <gtest/gtest.h>
+#include <t8_cmesh.h>
+#include <t8_cmesh/t8_cmesh_examples.h>
+
+/* Test for one tree*/
+/* *INDENT-OFF* */
+class cmesh_face_boundary_one_tree : public testing::TestWithParam<t8_eclass>{
+protected:
+  void SetUp() override {
+    eclass = GetParam();
+
+    /* For each eclass create a cmesh consisting only of one tree.
+     * We then check whether all faces of this tree are a boundary face. */
+    cmesh = t8_cmesh_new_from_class (eclass, sc_MPI_COMM_WORLD);
+  }
+  void TearDown() override {
+    t8_cmesh_destroy(&cmesh);
+  }
+
+  t8_cmesh_t          cmesh;
+  t8_eclass           eclass;
+  int                 num_faces;
+  int                 iface;
+};
+
+TEST_P (cmesh_face_boundary_one_tree, check_face_is_boundary_one_tree) {
+  
+    EXPECT_TRUE(t8_cmesh_is_committed (cmesh));
+    /* We now check each face */
+    num_faces = t8_eclass_num_faces[(int) eclass];
+    for (iface = 0; iface < num_faces; ++iface) {
+      EXPECT_TRUE(t8_cmesh_tree_face_is_boundary (cmesh, 0, iface));
+      EXPECT_TRUE(t8_cmesh_get_face_neighbor (cmesh, 0, iface, NULL, NULL) < 0);
```

Comment:
```suggestion
      EXPECT_LT(t8_cmesh_get_face_neighbor (cmesh, 0, iface, NULL, NULL), 0);
```
Have a look at:  https://github.com/google/googletest/blob/main/docs/reference/assertions.md
to find all the variety of EXPTECT/ASSERTION calls. You can even define your own ;)